Code/connect/include/Server.hpp

```diff
@@ -16,7 +16,7 @@ namespace TiltedPhoques
 
         TP_NOCOPYMOVE(Server);
 
-        bool Host(uint16_t aPort, uint32_t aTickRate) noexcept;
+        bool Host(uint16_t aPort, uint32_t aTickRate, bool bEnableDualStackIP = false) noexcept;

```

Comment:
If the bool is const in the source file it should be const in the header file, but i don't see a need for constness to be honest.

Reply:
Understood